README.md

```diff
@@ -77,6 +77,10 @@ Also check [Before you install](https://docs.bigbluebutton.org/2.3/install.html#
 | `bbb_monitoring_all_in_one_htpasswd_user` | The user for the htpasswd - _(required)_ if external | `Undefined` | |
 | `bbb_monitoring_all_in_one_htpasswd` | The password for the htpasswd - _(required)_ if external | `Undefined` | |
 | `bbb_monitoring_recordings_from_disk` | Collect recordings metrics by querying the disk instead of the API. See [this](https://bigbluebutton-exporter.greenstatic.dev/exporter-user-guide/#optimizations) for details. | `true` |
+| `bbb_monitoring_systemd_enable` | Deploy monitoring as systemd service(not recommended) | `false` | Works only when `bbb_monitoring_all_in_one_enable` is `false` |
+| `bbb_monitoring_systemd_version` | Version of the (greenstatic/bigbluebutton-exporter)[https://github.com/greenstatic/bigbluebutton-exporter] | `HEAD` | Could be a branch or tag |
```

Comment:
I think it does not make sense to create an additional version-variable for this, as `bbb_monitoring_all_in_one_version` exsits for the very same purpose (version of bbb-exporter).
Given the fact, that "all-in-one"-mode and "systemd"-mode mutually exclude each other, I would rather suggest to rename `bbb_monitoring_all_in_one_version` into `bbb_monitoring_bigblueblutton_exporter_version` or something like that. This could be used in both cases then.

Caveat of a shared variable is, that one would need to distinguish between "latest" and "HEAD" in the tasks.
But its simpler in the "frontend" (defaults/main.yml).

Reply:
The disadvantages outweigh and makes the Ansible code not better readable.

Reply:
@crosscodr What do you think about the following example for the default file?
```
bbb_monitoring_exporter_version: {% if bbb_monitoring_systemd_enable | bool %}HEAD{% endif %}{% if bbb_monitoring_all_in_one_enable | bool %}master{% endif %}
```

Reply:
Good idea, but jinja can be more elegant:
`bbb_monitoring_exporter_version: {{ 'HEAD' if bbb_monitoring_systemd_enable | bool else 'latest' }}`
I think we can leave the version to the current default if bbb_monitoring_systemd_enable is false.

Reply:
Thanks @crosscodr, that works